src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -78,13 +80,13 @@ export const styles = StyleSheet.create({
     color: COLORS.anon_primary
   },
   dateCreate: {
-    marginLeft: 20,
+    marginLeft: dimen.normalizeDimen(20),
     fontSize: normalizeFontSize(14),
     fontFamily: fonts.inter[400],
     lineHeight: normalizeFontSize(16.94),
     color: COLORS.black,
-    marginTop: 4,
-    marginBottom: 9
+    marginTop: dimen.normalizeDimen(4),
+    marginBottom: dimen.normalizeDimen(9)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `dimen.normalizeDimen` function for setting dimensions is a good practice as it helps to maintain consistency across different screen sizes and densities.
```diff
-    marginLeft: 20,
-    marginTop: 4,
-    marginBottom: 9
+    marginLeft: dimen.normalizeDimen(20),
+    marginTop: dimen.normalizeDimen(4),
+    marginBottom: dimen.normalizeDimen(9)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->